### ISSUE TITLE
Paddy stops working in automatic mode when I open another Sketch file

### ISSUE THREAD

**Issue by dMore88** [DWilliames/paddy-sketch-plugin#106]
This is an issue template.

Please fill in the relevant fields – whether this is a **Feature request** or a **Bug report**
Remove any sections that are not relevant to you.

---

#### Expected behavior:
When having two or more Sketch files opened Paddy auto-resize feature works fine as expected.

#### Actual behavior:
(Please include a screenshot or screen recording if possible)
When I open a file Paddy works perfect, but when I open another file, I mean having two Sketch files working at the same time, the auto resize feature stops working, I have to close Sketch then reopen it to have Paddy working again.

#### Steps to reproduce:
(Please attach a Sketch file if possible)
- Open Sketch File, or create a new one
- Create a group or whatever layer with a background as Paddy requests ([x x x x])
- Paddy auto-resize works fine
- Open or create another Sketch file with the previous one still opened
- Do something in the new file
- Paddy auto-resize feature stops working
- Close Sketch
- Reopen Sketch
- Paddy auto-resize works again.

**Comment by marysea** [comment]
Yep… this is exactly what I'm experiencing. I thought I was going mad – then I followed this and hey presto, works if only one file open… at a time.
